ibis/expr/operations.py

```diff
@@ -139,7 +139,7 @@ def root_tables(self):
         return distinct_roots(*exprs)
 
     def resolve_name(self):
-        raise com.ExpressionError('Expression is not named: %s' % repr(self))
+        raise com.ExpressionError('Expression is not named: %s' % type(self))
```

Comment:
Might as well move this to an f-string here since you're touching this code.

Reply:
Yes good point, updated